hindsight/io_modules/derived_stream/redshift.lua

```diff
@@ -0,0 +1,64 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+local M = {}
+local tostring  = tostring
+local type      = type
+
+local date      = require "os".date
+local floor     = require "math".floor
+local gsub      = require "string".gsub
+
+setfenv(1, M) -- Remove external access to contain everything in the module
+
+VARCHAR_MAX_LENGTH = 65535
+
+function strip_nonprint(v)
```

Comment:
We need automated tests for all the lua code (that is a small project in itself)